partner_contact_nutrition_allergen/models/res_partner.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from openerp import models, fields
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    nutrition_allergen_ids = fields.Many2many('product.product',
+                                              'res_partner_allergen_rel',
+                                              'partner_id',
+                                              'product_id',
```

Comment:
Any reason for explicitly defining these? I could be wrong, but I think the guidelines say it's best to leave this automatic unless there's an issue.